main.tf

```diff
@@ -3,7 +3,7 @@ terraform {
     # for the infra that will host Psoxy instances
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 4.12"
+      version = "~> 4.22"
```

Comment:
np: It that expected ? Just double-checking.

Reply:
yes, thank you; 4.12 actually does NOT work; this is probably something we should figure out a better way to automatically test.  Our usual tests are in effect validating that the most recent version satisfying the constraints work, rather than than the oldest version matching the constraint.